### DISCUSSION TITLE
Broken link issue

### DISCUSSION THREAD

**Discussion by Nabeel-javed** [solodit/feedback#20] [Bug Report] [answered]
Hey guys. I'm facing broken link issue on solidit. I've coppied the link from the issue but when i open the link its broken and not working

**Comment by hans-solodit** [comment]
Could you let us know the link you are trying?

**Reply by Nabeel-javed** [reply]
https://solodit.xyz/issues/18255
https://solodit.xyz/issues/5699
These two links

**Reply by hans-cyfrin** [reply, marked as answer]
Those issues were removed as we developed new features. 
To prevent this in the future, we've now transitioned to using slug formatted URLs.
We appreciate your vigilance in bringing these issues to our attention.
